From 562a5f3e7a54e1b0d6918df355710882366cbcb8 Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Wed, 7 Jun 2017 20:46:18 -0400
Subject: [PATCH 1146/1936] Microtonal: Fix Octave Size When Loading .SCL

When applying microtonal settings, an octave size of 12 would be
forced if microtonal support was disabled. This caused some incorrect
loading in previous versions.
---
 src/Misc/Microtonal.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Misc/Microtonal.cpp b/src/Misc/Microtonal.cpp
index 1560eb0b..87300e40 100644
--- a/src/Misc/Microtonal.cpp
+++ b/src/Misc/Microtonal.cpp
@@ -851,7 +851,7 @@ void Microtonal::apply(void)
     {
         char buf[100*MAX_OCTAVE_SIZE] = {0};
         char tmpbuf[100] = {0};
-        for (int i=0;i<getoctavesize();i++){
+        for (int i=0;i<octavesize;i++){
             if (i!=0)
                 strncat(buf, "\n", sizeof(buf)-1);
             tuningtoline(i,tmpbuf,100);
-- 
2.47.0

